cpanfile

```diff
@@ -0,0 +1,13 @@
+requires 'Locale::Country';
+requires 'Locale::Country::Multilingual';
+requires 'perl', '5.006';
+
+on configure => sub {
+    requires 'ExtUtils::MakeMaker';
```

Comment:
Would be useful here if we're depending on a specific minimum MakeMaker version; otherwise this can be dropped

Reply:
this cpanfile is generated auto by ` mymeta-cpanfile > cpanfile`
should we change it manually?

Reply:
Yes, `mymeta-cpanfile` is a transition tool mostly, maintainers should be updating `cpanfile` directly.

Reply:
updated